### PR TITLE
Add Translation doc

### PR DESCRIPTION
**Pull request by mandyedi**
A translation process guide for #56 
This is the very first version of the guide. It might change as we start to implement the translation and find out the actual tricks about it.
I can start to work on it the next week.

### CODE REVIEW

File: doc/Translation.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+{
+    QLabel *label = new QLabel(
+                QCoreApplication::translate("LoginWidget", "Password:"), logwid);
+}
+```
+
+
+### Generating Translation Files
+
+If new language is added to the project or new phrases (QLabels, any literals taht apperas on UI) added to the source code, run ```lupdate NotepadNext.pro```.
```
[the remark this answers is not in the document]
Comment (mandyedi) [reply doc/Translation.md:64]: Great, thank you!

**Comment by mandyedi** [comment]
The translation doc is ready for review.

**Comment by dail8859** [comment]
@mandyedi Thanks so much for putting this together. I glanced through it real quick and it looks good. Glad to see its separated out between developer and translator. Ideally someone that wants to translate the application doesn't need to know about the underlying details (unless they are curious of course).

I think for now I'll wait to merge this until #103 is completed as there might be some updates to the document, etc.

**Comment by mandyedi** [comment]
Okay, thank you!

**Comment by dail8859** [comment]
@mandyedi Sorry this took me a while. Been super busy and finally got around to revising this. I took out alot of the specific information since it is documented very well on other websites. 

Thanks for the PR!

**Pull request merged**